TIGLViewer/src/TIGLViewerShapeIntersectionDialog.ui

```diff
@@ -180,6 +186,195 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="tabSPS">
```

Comment:
The new tab of the shape intersection dialog does not yet scale, when I resize the dialog.
Please use the layouts to make the ui elements resizable. Have a look onto the other tabs, which resize with the window.